data/json/items/melee/swords_and_blades.json

```diff
@@ -1101,6 +1101,41 @@
     "techniques": [ "PRECISE", "RAPID", "WBLOCK_2" ],
     "flags": [ "STAB", "SHEATH_SWORD" ]
   },
+  {
+    "id": "sword_cane",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "brown",
+    "name": "sword cane (sheathed)",
+    "description": "A cane containing a hidden blade.  The sword cane was a popular fashion accessory for the wealthy during the 18th and 19th centuries.  Activate to unsheath the blade.",
+    "price": 2000,
+    "price_postapoc": 1000,
+    "material": [ "steel", "wood" ],
+    "weight": 650,
+    "volume": "1531 ml",
+    "bashing": 3,
+    "to_hit": 2,
+    "use_action": { "type": "transform", "target": "sword_cane_on", "msg": "You unsheath your sword cane." },
+    "techniques": [ "PRECISE", "RAPID", "WBLOCK_2" ]
+  },
+  {
+    "id": "sword_cane_on",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "sword cane",
+    "description": "A cane containing a hidden blade.  The sword cane was a popular fashion accessory for the wealthy during the 18th and 19th centuries.  Activate to sheath the blade.",
+    "price": 2000,
+    "price_postapoc": 1000,
+    "material": [ "steel", "wood" ],
+    "weight": 650,
+    "volume": "1200 ml",
```

Comment:
Volume maybe should go up instead of down to keep the cane sheath part modeled somehow

Reply:
No because it will affect the speed as if you were handling a weapon twice the volume of the sword, and it doesn't make much sense.